### ISSUE TITLE
allow for reads to be mapped form other organisms

### ISSUE THREAD

**Issue by bradfordcondon** [tripal/tripal_analysis_expression#155]
We want Biosamples to come from organisms that dont match the feature, and expression reads to be mapped from said organism.

**Comment by jhumann** [comment]
Yes, please!  I am trying to load expression data from grapefruit that was mapped to the clementine genome.  I will have a ton of data on the Citrus Genome Database where the reads are from a different organism and mapped to the genome of another species.

Jodi
